lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -859,24 +880,50 @@ llvm::Optional<std::string> SwiftLanguageRuntimeImpl::GetEnumCaseName(
   if (eti->projectEnumValue(*GetMemoryReader(), addr, &case_index))
     return eti->getCases()[case_index].Name;
 
+  if (eti->projectEnumValue(*GetMemoryReader(), addr, &case_index))
+    return eti->getCases()[case_index].Name;
+
+  // FIXME: Enabling this fails TestSwiftNestedCEnums.py: The test
+  // nests a C-style enum inside of a payload-carrying enum and most
+  // likely we're not stripping off the outer enum's discriminator
+  // before reading the value of the inner one.
+  
+  // LogUnimplementedTypeKind(__FUNCTION__, type);
   return {};
 }
 
-std::pair<bool, llvm::Optional<size_t>>
+static bool IsSIMDType(CompilerType type) {
```

Comment:
In fact, there's already a `IsSIMDNode` inside `TypeSystemSwiftTypeRef.cpp`, although it's only used for debugging. Maybe that function and this one should be coalesced?

Reply:
They're _not quite_ the same.